float64 delta_t

float64 sigma_w
float64 s_w
float64 sigma_z
float64 s_z

float64[] y_model
float64[] y_noise
float64[] y_model_noise
